repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -332,19 +341,27 @@ def _get_rhsm_available_repoids(context):
     #
     repoids = rhsm.get_available_repo_ids(context)
     if not repoids or len(repoids) < 2:
-        raise StopActorExecutionError(
-            message='Cannot find required basic RHEL {} repositories.'.format(target_major_version),
-            details={
-                'hint': ('It is required to have RHEL repositories on the system'
-                         ' provided by the subscription-manager unless the --no-rhsm'
-                         ' options is specified. Possibly you'
-                         ' are missing a valid SKU for the target system or network'
-                         ' connection failed. Check whether your system is attached'
-                         ' to a valid SKU providing RHEL {} repositories.'
-                         ' In case the Satellite is used, read the upgrade documentation'
-                         ' to setup the satellite and the system properly.'.format(target_major_version))
-            }
-        )
+        reporting.create_report([
+            reporting.Title('Cannot find required basic RHEL target repositories.'),
+            reporting.Summary(
+                'During the configuration for the target system, no enabled repositories were found.'
+                ' This can happen when a repository ID was entered incorrectly either while using the --enablerepo'
+                ' option of leapp or in a third party actor that produces a CustomTargetRepositoryMessage.'
+            ),
+            reporting.Tags([reporting.Tags.REPOSITORY]),
+            reporting.Flags([reporting.Flags.INHIBITOR]),
+            reporting.Remediation(hint=(
+                'It is required to have RHEL repositories on the system'
+                ' provided by the subscription-manager unless the --no-rhsm'
+                ' option is specified. You might be missing a valid SKU for'
+                ' the target system or have a failed network connection.'
+                ' Check whether your system is attached to a valid SKU that is'
+                ' providing RHEL {} repositories.'
+                ' If you are using Red Hat Satellite, read the upgrade documentation'
+                ' to set up Satellite and the system properly.'
```

Comment:
as mentioned in original PR in comments, we should skip all "you" etc.. E.g.:
- "You might be missing" -> A valid SKU for the target system could be missing or the network connection might failed...